dbcon/mysql/ha_mcs_execplan.cpp

```diff
@@ -9123,7 +9153,7 @@ int getGroupPlan(gp_walk_info& gwi, SELECT_LEX& select_lex, SCSEP& csep, cal_gro
         funcFieldVec[i]->print(&str, QT_ORDINARY);
         sc->alias(string(str.c_ptr()));
         //sc->tableAlias(funcFieldVec[i]->table_name);
-        sc->tableAlias(sc->tableAlias());
+        sc->tableAlias(sc->alias());
```

Comment:
This should have `lower_case_table_names` as the second argument.

Reply:
Anything put into SimpleColumn has been converted if needed. No need to spend cpu cycles doing it again.